faq/ppv.php

```diff
@@ -20,13 +20,25 @@
 <h2>Contents</h2>
 
 <p>
-<a href="#gen">A) General Procedures</a><br>
-<a href="#rr">B) Requests to Revise and Resubmit</a><br>
+<a href="#general">A) General Procedures</a><br>
+<a href="#revise">B) Requests to Revise and Resubmit</a><br>
 <a href="#checks">C) Specific Checks to Make</a><br>
 <a href="#printers">D) Printers' Errors and Transcriber's Note</a><br>
 <a href="#reader">E) Checking E-reader versions</a><br>
 <a href="#rptcard">F) Submitting a PPV Summary</a><br>
 <a href="#late">G) Late PPV Summaries</a>
+/* The following are dummy IDs to provide a way to redirect to the
```

Comment:
You don't need any of these.

Reply:
Ohhhh ... I misunderstood, then.

Reply:
All changes from both `faq/ppv.php` and `faq/post_proof.php` were unnecessary and have been reverted, and the commit they were in squashed. I didn have to change one link in `tools/post_proofers/ppv_report.php` to match the reverted link.